src/interface/blas2_interface.hpp

```diff
@@ -417,74 +417,67 @@ typename sb_handle_t::event_t _symv_impl(
 }
 
 /*! _gbmv_impl.
- * @brief Implementation of the Band Matrix Vector product.
+ * @brief Implementation of the Generic Band Matrix Vector product.
  *
  */
 template <uint32_t local_range, transpose_type trn, typename sb_handle_t,
           typename index_t, typename element_t, typename container_t0,
           typename container_t1, typename increment_t, typename container_t2>
-typename sb_handle_t::event_t _gbmv_impl(sb_handle_t& sb_handle, char _trans,
-                                         index_t _M, index_t _N, index_t _KL,
-                                         index_t _KU, element_t _alpha,
-                                         container_t0 _mA, index_t _lda,
-                                         container_t1 _vx, increment_t _incx,
-                                         element_t _beta, container_t2 _vy,
-                                         increment_t _incy) {
-  constexpr bool is_transposed = (trn != transpose_type::Normal);
-
+typename sb_handle_t::event_t _gbmv_impl(sb_handle_t& sb_handle, index_t _M,
+                                         index_t _N, index_t _KL, index_t _KU,
+                                         element_t _alpha, container_t0 _mA,
+                                         index_t _lda, container_t1 _vx,
+                                         increment_t _incx, element_t _beta,
+                                         container_t2 _vy, increment_t _incy) {
   if ((_KL >= _M) || (_KU >= _N)) {
-    throw std::invalid_argument("Erroneous parameter");
+    throw std::invalid_argument("Erroneous parameter: _KL >= _M || _KU >= _N");
   }
 
+  constexpr bool is_transposed = (trn != transpose_type::Normal);
```

Comment:
Do the specializations work for conjugate transpose?

Reply:
Thanks for your review, @hjabird!

Yes, in case of *conjugate transpose* for real numbers it acts like *transposed* as stated [here](https://spec.oneapi.io/versions/latest/elements/oneMKL/source/architecture/architecture.html#onemkl-datatypes).